sql panic: unhandled node type: *sql.commentOnTableNode

https://teamcity.cockroachdb.com/viewLog.html?buildId=1239637&tab=buildResultsDiv&buildTypeId=Cockroach_Nightlies_SqlLogicTestHighVModuleNightly

Also mentioned in https://github.com/cockroachdb/cockroach/issues/35660#issuecomment-473893440 so it's been there for a while

```
panic: unhandled node type: *sql.commentOnTableNode [recovered]
	panic: panic while executing 1 statements: COMMENT ON TABLE _ IS 'waa'; caused by unhandled node type: *sql.commentOnTableNode

goroutine 15368 [running]:
github.com/cockroachdb/cockroach/pkg/sql.(*connExecutor).closeWrapper(0xc00162c000, 0x3156940, 0xc000af7700, 0x26c6340, 0xc001a66c50)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/conn_executor.go:719 +0x36d
github.com/cockroachdb/cockroach/pkg/sql.(*Server).ServeConn.func1(0xc00162c000, 0x3156940, 0xc000af7700)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/conn_executor.go:431 +0x61
panic(0x26c6340, 0xc001a66c50)
	/usr/local/go/src/runtime/panic.go:513 +0x1b9
github.com/cockroachdb/cockroach/pkg/sql.planPhysicalProps(0x31578c0, 0xc001a66bb0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/plan_physical_props.go:144 +0x1491
github.com/cockroachdb/cockroach/pkg/sql.planToString(0x3156a00, 0xc0016e6750, 0x31578c0, 0xc001a66bb0, 0x0, 0x0, 0x0, 0x0, 0x31578c0)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/explain_plan.go:378 +0x338
github.com/cockroachdb/cockroach/pkg/sql.(*planner).makePlan(0xc00162c350, 0x3156a00, 0xc0016e6750, 0x2abc705, 0x12)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/plan.go:378 +0x59d
github.com/cockroachdb/cockroach/pkg/sql.(*connExecutor).makeExecPlan(0xc00162c000, 0x3156a00, 0xc0016e6750, 0xc00162c350, 0x10, 0x89ebff)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/conn_executor_exec.go:1003 +0x305
github.com/cockroachdb/cockroach/pkg/sql.(*connExecutor).dispatchToExecutionEngine(0xc00162c000, 0x3156a00, 0xc0016e6750, 0xc00162c350, 0x7f3674c4d2b0, 0xc0030102c0, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/conn_executor_exec.go:865 +0x148
[23:28:02] :						 [/regression_32834] [Test Output]
github.com/cockroachdb/cockroach/pkg/sql.(*connExecutor).execStmtInOpenState(0xc00162c000, 0x3156a00, 0xc0016e6750, 0x315ae80, 0xc004182840, 0xc0009834af, 0x1b, 0x0, 0x0, 0x0, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/conn_executor_exec.go:456 +0xdd0
github.com/cockroachdb/cockroach/pkg/sql.(*connExecutor).execStmt(0xc00162c000, 0x3156a00, 0xc0016e6750, 0x315ae80, 0xc004182840, 0xc0009834af, 0x1b, 0x0, 0x0, 0x0, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/conn_executor_exec.go:102 +0x610
github.com/cockroachdb/cockroach/pkg/sql.(*connExecutor).run(0xc00162c000, 0x3156940, 0xc000af7700, 0xc00090b9b8, 0x5400, 0x15000, 0xc00090ba50, 0xc0009daf60, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/conn_executor.go:1182 +0x21c8
github.com/cockroachdb/cockroach/pkg/sql.(*Server).ServeConn(0xc0006a7b80, 0x3156940, 0xc000af7700, 0xc00162c000, 0x5400, 0x15000, 0xc00090ba50, 0xc0009daf60, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/conn_executor.go:433 +0xce
github.com/cockroachdb/cockroach/pkg/sql/pgwire.(*conn).serveImpl.func4(0xc0006a7b80, 0x3156940, 0xc000af7700, 0xc00075c400, 0xc000306e44, 0xc0009daf60, 0xc00162c000, 0x5400, 0x15000, 0xc00090ba50, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/pgwire/conn.go:337 +0xe6
created by github.com/cockroachdb/cockroach/pkg/sql/pgwire.(*conn).serveImpl
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/pgwire/conn.go:320 +0x1080
```

--- a/pkg/sql/plan_physical_props.go
+++ b/pkg/sql/plan_physical_props.go
@@ -140,6 +140,8 @@ func planPhysicalProps(plan planNode) physicalProps {
 	case *valuesNode:
 	case *virtualTableNode:
 	case *zeroNode:
+	case *commentOnTableNode:
+	case *commentOnColumnNode:
 	default:
 		panic(fmt.Sprintf("unhandled node type: %T", plan))
 	}

